.ci/README.md

```diff
@@ -334,3 +336,13 @@ To do so:
 7. In the input parameters form, set the `GITHUB_CHECK_SHA1` to the `SHA1` of the last commit in your pull request. This value will allow us to modify the mergeable status of that commit with the Github check. Besides that, it will set the specific directory in the GCP bucket to look up the CI binaries.
 8. In the input parameters form, set the `GITHUB_CHECK_REPO` to `elastic-agent` or `beats`, depending where the aforementioned SHA1 belongs. This is important to look up the binaries in the right GCP bucket.
 9. Click the `Build` button at the bottom of the parameters form.
+
+##Troubleshooting
+
+Make sure :
+- Python 3.9 or higher is installed.
+- Install python3-venv to resolve virtual environment creation errors.
+- In case, you are getting errors while creating AWS VMs, ensure you have exported AWS secret and access ids.
+- Check 600 permission is provided to id_rsa key files.
+- Run list-platforms command and export Node variabe to resolve Node creation errors.
+- While creating windows node, we need to run `create-node` command in portions such as `provision-node` and `setup-node`. Also, some times you need to ssh node to create it successfully.
```

Comment:
I'd add that it's caused because Windows instances takes a lot of time to be created on AWS, so that it's clearer